scrapers/WankzVR.yml

```diff
@@ -0,0 +1,25 @@
+name: "WankzVR"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - wankzvr.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    common:
+      $info: //div[@class="detail"]
+    scene:
+      Title: $info//h1/text()
```

Comment:
Title gets matched twice.
`Title: //div[@class="detail__header detail__header-lg"]/h1` seems to work